src/main/java/com/gamingmesh/jobs/CMILib/ItemManager.java

```diff
@@ -731,9 +731,9 @@ public static String getRealNameByType(EntityType type) {
 	CARROT(391, 0, 22824, "Carrot", "Carrotitem"),
 	CARROTS(141, 0, 17258, "Carrots", "CARROT"),
 	CARROT_ON_A_STICK(398, 0, 27809, "Carrot on a Stick", "carrotstick"),
-	CARVED_PUMPKIN(-1, -1, 25833, "Carved Pumpkin", ""),
+	CARVED_PUMPKIN(91, 0, 25833, "Carved Pumpkin", "JACK_O_LANTERN"),
 	CAULDRON(380, 0, 26531, "Cauldron", "CAULDRON_ITEM"),
-	CAVE_AIR(-1, -1, 17422, "Air", ""),
+	CAVE_AIR(0, 0, 17422, "Air", "AIR"),

```

Comment:
CAVE_AIR and CARVED_PUMPKIN sub-ID is invalid. CARVED_PUMPKIN is not JACK_O_LANTERN.